pyaedt/edb_core/edb_data/hfss_simulation_setup_data.py

```diff
@@ -0,0 +1,1700 @@
+        self.mesh_operation.RefineInside = value
+        self._parent._update_setup()
+
+
+class HfssPortSettings(object):
+    """Manages EDB methods for hfss port settings."""
+
+    def __init__(self, parent):
+        self._parent = parent
+
+    @property
+    def _hfss_port_settings(self):
+        return self._parent._edb_sim_setup_info.SimulationSettings.HFSSPortSettings
+
+    @property
+    def max_delta_z0(self):
+        """Maximum change to Z0 in successive passes.
+
+        Returns
+        -------
+        float
+        """
+        return self._hfss_port_settings.MaxDeltaZ0
+
+    @max_delta_z0.setter
+    def max_delta_z0(self, value):
+        self._hfss_port_settings.MaxDeltaZ0 = value
+        self._parent._update_setup()
+
+    @property
+    def max_triangles_wave_port(self):
+        """Max number of triangles allowed for wave ports.
+
+        Returns
+        -------
+        int
+        """
+        return self._hfss_port_settings.MaxTrianglesWavePort
+
+    @max_triangles_wave_port.setter
+    def max_triangles_wave_port(self, value):
+        self._hfss_port_settings.MaxTrianglesWavePort = value
+        self._parent._update_setup()
+
+    @property
+    def min_triangles_wave_port(self):
+        """Minimum number of triangles allowed for wave ports.
+
+        Returns
+        -------
+        int
+        """
+        return self._hfss_port_settings.MinTrianglesWavePort
+
+    @min_triangles_wave_port.setter
+    def min_triangles_wave_port(self, value):
+        self._hfss_port_settings.MinTrianglesWavePort = value
+        self._parent._update_setup()
+
+    @property
+    def enable_set_triangles_wave_port(self):
+        """Enable setting of min/max mesh limits for wave ports.
+
+        Returns
+        -------
+        bool
+        """
+        return self._hfss_port_settings.SetTrianglesWavePort
+
+    @enable_set_triangles_wave_port.setter
+    def enable_set_triangles_wave_port(self, value):
+        self._hfss_port_settings.SetTrianglesWavePort = value
+        self._parent._update_setup()
+
+
+class HfssSolverSettings(object):
+    """Manages EDB methods for hfss solver settings."""
+
+    def __init__(self, parent):
+        self._parent = parent
+
+    @property
+    def _hfss_solver_settings(self):
+        return self._parent._edb_sim_setup_info.SimulationSettings.HFSSSolverSettings
+
+    @property
+    def enhanced_low_freq_accuracy(self):
+        """Enable legacy low frequency sampling.
+
+        Returns
+        -------
+        float
+        """
+        return self._hfss_solver_settings.EnhancedLowFreqAccuracy
+
+    @enhanced_low_freq_accuracy.setter
+    def enhanced_low_freq_accuracy(self, value):
+        self._hfss_solver_settings.EnhancedLowFreqAccuracy = value
+        self._parent._update_setup()
+
+    @property
+    def order_basis(self):
+        """Get or Specify order of the basis functions for HFSS : 0=Mixed, 1=Zero, 2=1st order, 3=2nd order."""
```

Comment:
```suggestion
        """Order of the basic functions for HFSS:
        - 0=Mixed
        - 1=Zero
        - 2=1st order
        - 3=2nd order"""
```
Alex said that examples should show usage.